pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -128,6 +128,12 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 			return err
 		}
 	}
+	if nodeVersionInfoUpdate.HasMajorOrMinorUpdate() {
```

Comment:
This is the same condition as the one below, why not combine them?

Reply:
Yes, my bad. This was the basis for a convo. The PR is not yet ready, I didn't even test it.

Reply:
What if somebody will start to upgrade from CaaSP 4.0 GM, where there is `crio v1.15` ?

Reply:
Yes the code is wrong, needs the guard @cmurphy was mentioning.
Should be fixed now.